resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -322,7 +322,8 @@ private[spark] class KubernetesClusterSchedulerBackend(
             .addToRequests("cpu", executorCpuQuantity)
             .addToLimits("cpu", executorCpuQuantity)
           .endResources()
-          .withEnv(requiredEnv.asJava)
+          .addAllToEnv(requiredEnv.asJava)
+          .addToEnv(executorExtraClasspathEnv.toSeq: _*)
```

Comment:
they're not running right now, but I can temporarily re-enable, you push a commit, and we watch them run to see if they would've caught this